frontend/src/pages/Dashboard/index.css

```diff
@@ -1,9 +1,26 @@
 @import '~antd/dist/antd.css';
```

Comment:
![Screenshot 2021-03-12 at 10 40 53 AM](https://user-images.githubusercontent.com/15646062/110884064-6ebc6f80-831f-11eb-8e08-2def59d8f57e.png)

If the backend does not started, the frontend should popup a message to tell user. Not infinite loop.

Reply:
I'll set a timeout for get request to avoid this